server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -645,6 +663,77 @@ private Optional<FormTag> optionalToggle(
             .with(optionalButton));
   }
 
+  private Optional<FormTag> addressCorrectionEnabledToggle(
+      Request request,
+      InputTag csrfTag,
+      long programDefinitionId,
+      long blockDefinitionId,
+      QuestionDefinition questionDefinition,
+      boolean addressCorrectionEnabled) {
+    if (!questionDefinition.isAddress()) {
+      return Optional.empty();
+    }
+    if (questionDefinition instanceof StaticContentQuestionDefinition) {
+      return Optional.empty();
+    }
```

Comment:
question (blocking): is this necessary given the `isAddress` check above it?

Reply:
Good catch. Removed the check.